conf/machine/raspberrypi5.conf

```diff
@@ -18,7 +18,7 @@ RPI_KERNEL_DEVICETREE = " \
 "
 
 SDIMG_KERNELIMAGE ?= "kernel_2712.img"
-SERIAL_CONSOLES ?= "115200;ttyS0"
+SERIAL_CONSOLES ?= "115200;ttyAMA10
```

Comment:
@floion following your git rebase of my original git commit a quote (") is missing at the end of the line. This leads to a parse error. It should be like: https://github.com/leon-anavi/meta-raspberrypi-1/commit/c2ce56df13b43a90e170951dbe60479a84c1b413